src/main/java/com/thirdcc/webapp/repository/EventRepository.java

```diff
@@ -17,4 +17,6 @@
 public interface EventRepository extends JpaRepository<Event, Long> {
 
     Optional<Event> findOneByIdAndStatusIn(Long id, Set<EventStatus> eventStatus);
+
+    Optional<Event> findOneById(Long id);
```

Comment:
@junzai97  no need to define this method
just use findById that is auto generated
you make a same mistake again 😆

Reply:
Issue resolved in 3f4cfb491c07b0cb21cc1cb7bdd9dfa6709c27d5